handlers/desired_lrp_handlers.go

```diff
@@ -489,3 +505,32 @@ func (h *DesiredLRPHandler) updateInstances(ctx context.Context, logger lager.Lo
 		}
 	}
 }
+
+func (h *DesiredLRPHandler) logDesiredLrpParsingErrors(err *models.Error, processGuid string) error {
+	if err == nil {
+		return errors.New("parsing error is nil")
+	}
+
+	appGuid := parseAppGuidFromProcessGuid(processGuid)
+	if appGuid == "" {
+		return errors.New("app guid is empty")
```

Comment:
**Question**

Is it possible to ever get an empty app guid?

Reply:
It's possible if `parseAppGuidFromProcessGuid` cant parse it from the ProcessGUID, which can happen if the ProcessGUID field in the request fails validation.

Reply:
And is there a situation where process GUID is invalid?

Reply:
Not sure, but even if it's not I need to ensure its valid before I process it so it doesn't panic with `slice bounds out of range` in `parseAppGuidFromProcessGuid`